e2e/tests/interchain_accounts/utils.go

```diff
@@ -0,0 +1,12 @@
+package interchainaccounts
+
+import (
+	icatypes "github.com/cosmos/ibc-go/v8/modules/apps/27-interchain-accounts/types"
+)
+
+func DefaultTestMetadataVersionString(controllerConnectionID, hostConnectionID string) string {
```

Comment:
I'd like to pass an empty version string in the e2e test to ensure it works. So this function should not be needed and you can just use `NewDefaultMetadataString`

Reply:
I didn't mean you should change the function name. I meant that you could use the `NewDefaultMetadataString` function from icatypes instead of defining a new function here @GNaD13

Reply:
I got it

Reply:
@srdtrk about the e2e test. If we use icatypes NewDefaultMetadataString with only controller port ID, it may get an error when ChanOpenInit check [here](https://github.com/decentrio/ibc-go/blob/ad3e34427557fc08f97caa01b727dc34a0d9317d/modules/apps/27-interchain-accounts/controller/keeper/handshake.go#L64). Should we use empty version so controller chain could create [metadata](https://github.com/decentrio/ibc-go/blob/ad3e34427557fc08f97caa01b727dc34a0d9317d/modules/apps/27-interchain-accounts/controller/keeper/handshake.go#L57) itself?

Reply:
Oh, I see the issue now. Sorry about my oversight. We can go back to using your `DefaultTestMetadataVersionString`. I'll approve this PR after the e2e are passing